### ISSUE TITLE
ci-trial #3 failed

### ISSUE THREAD

**Issue by biaomingzhong** [biaomingzhong/ci-trial#2]
Build 'ci-trial' is failing!

Last 50 lines of build output:

```
Started by GitHub push by biaomingzhong
Building in workspace /Users/mos/.jenkins/workspace/ci-trial
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/biaomingzhong/ci-trial.git # timeout=10
Fetching upstream changes from https://github.com/biaomingzhong/ci-trial.git
 > git --version # timeout=10
using GIT_SSH to set credentials 
 > git fetch --tags --progress https://github.com/biaomingzhong/ci-trial.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision ca09f9bceb985ac51a17df6e87fa83f7a0666c45 (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f ca09f9bceb985ac51a17df6e87fa83f7a0666c45
Commit message: "线上环境提交测试错误的build的时候自动创建issues"
 > git rev-list 45e480da2053d1e47c0066227492e454330a0b28 # timeout=10
[ci-trial] $ /usr/local/bin/zsh -xe /var/folders/jz/3chsghpj1xvb6dmwmg_tl7nr0000gn/T/jenkins8202385504792030465.sh
+/var/folders/jz/3chsghpj1xvb6dmwmg_tl7nr0000gn/T/jenkins8202385504792030465.sh:2> chmod 744 ./jenkins/test.sh
[Set GitHub commit status (universal)] PENDING on repos [GHRepository@1459eac6[description=<null>,homepage=<null>,name=ci-trial,license=<null>,fork=false,size=0,milestones={},language=Groovy,commits={},source=<null>,parent=<null>,responseHeaderFields={null=[HTTP/1.1 200 OK], Access-Control-Allow-Origin=[*], Access-Control-Expose-Headers=[ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval], Cache-Control=[private, max-age=60, s-maxage=60], Content-Encoding=[gzip], Content-Security-Policy=[default-src 'none'], Content-Type=[application/json; charset=utf-8], Date=[Thu, 16 Nov 2017 10:51:06 GMT], ETag=[W/"5a4081d166c401ae1ca7506b5de67a43"], Last-Modified=[Thu, 16 Nov 2017 10:28:08 GMT], OkHttp-Received-Millis=[1510829466531], OkHttp-Response-Source=[CONDITIONAL_CACHE 200], OkHttp-Selected-Protocol=[http/1.1], OkHttp-Sent-Millis=[1510829466162], Server=[GitHub.com], Status=[200 OK], Strict-Transport-Security=[max-age=31536000; includeSubdomains; preload], Transfer-Encoding=[chunked], Vary=[Accept, Authorization, Cookie, X-GitHub-OTP, Accept-Encoding], X-Accepted-OAuth-Scopes=[repo], X-Content-Type-Options=[nosniff], X-Frame-Options=[deny], X-GitHub-Media-Type=[github.v3; format=json], X-GitHub-Request-Id=[C3C5:2078E:A001E5:CF34E9:5A0D6D9A], X-OAuth-Scopes=[admin:gpg_key, admin:org, admin:org_hook, admin:public_key, admin:repo_hook, delete_repo, gist, notifications, repo, user], X-RateLimit-Limit=[5000], X-RateLimit-Remaining=[4981], X-RateLimit-Reset=[1510832005], X-Runtime-rack=[0.050228], X-XSS-Protection=[1; mode=block]},url=https://api.github.com/repos/biaomingzhong/ci-trial,id=110957476]] (sha:ca09f9b) with context:Jenkins
Setting commit status on GitHub for https://github.com/biaomingzhong/ci-trial/commit/ca09f9bceb985ac51a17df6e87fa83f7a0666c45
[ci-trial] $ /usr/local/bin/zsh -xe /var/folders/jz/3chsghpj1xvb6dmwmg_tl7nr0000gn/T/jenkins3701973959347897573.sh
+/var/folders/jz/3chsghpj1xvb6dmwmg_tl7nr0000gn/T/jenkins3701973959347897573.sh:2> ./jenkins/test.sh failed
build success
develop 1.0.1 add line
jenkins env var change author  branch name  job name ci-trial
develop 1.0.2 add new line
develop 1.0.3 new task
develop 1.0.4 something went wrong
Build step 'Execute shell' marked build as failure

```

Changes since last successful build:
- [quinlanzhong] ca09f9bceb985ac51a17df6e87fa83f7a0666c45 - 线上环境提交测试错误的build的时候自动创建issues

[View full output](https://jenkins.mjb6.cn/job/ci-trial/3/)
